integrations/db-filedb/src/FileDb.ts

```diff
@@ -16,25 +8,33 @@ export interface FileDbConfig extends DbPluginConfig {
   primaryKeyColumn?: string;
 }
 
-export interface FileDbItem {
-  id: string;
-  user?: PersistableUserData;
-  session?: PersistableSessionData;
-  createdAt?: string;
-  updatedAt?: string;
-}
+export class FileDb extends DbPlugin<FileDbConfig> {
+  constructor(config?: FileDbConfig) {
+    super(config);
+  }
 
-export class FileDb extends Plugin<FileDbConfig> {
   getDefaultConfig(): FileDbConfig {
     return {
       pathToFile: './db/db.json',
```

Comment:
Was the v3 default `../db/db.json`? If I see it correctly, this would result in the `db` being stored in `src/db` instead of `db`

Reply:
Yes, `../db/db.json` would be probably clearer since `app.dev.ts` is in `/src`. 
Another option could be:
```
app.use(
  new FileDb({
    pathToFile: __dirname + '../db/db.json',
    })
);
```